content/docs/strict-mode.md

```diff
@@ -85,4 +86,12 @@ For example, consider the following code:
 
 At first glance, this code might not seem problematic. But if `SharedApplicationState.recordEvent` is not [idempotent](https://en.wikipedia.org/wiki/Idempotence#Computer_science_meaning), then instantiating this component multiple times could lead to invalid application state. This sort of subtle bug might not manifest during development, or it might do so inconsistently and so be overlooked.
 
-By intentionally double-invoking methods like the component constructor, strict mode makes patterns like this easier to spot.
\ No newline at end of file
+By intentionally double-invoking methods like the component constructor, strict mode makes patterns like this easier to spot.
+
+### Detecting legacy context APIs
+
+As is explained in the [previous legacy `context` post](/docs/legacy-context.html), the prior usage of context is error-prone, and will be removed in a future major version. It still works for all 16.x releases but will show up this warning message in strict mode:
```

Comment:
Nits:

- The first phrase (everything before "the prior usage of context" is redundant, since you also link to the context doc at the end of the section. I think you can remove it.
- Let's be consistent and refer to it as "legacy context API" instead of "the prior usage of context" and "new context API" instead of "new usage of context APIs"